src/react-ruler/Ruler.tsx

```diff
@@ -119,21 +122,16 @@ export default class Ruler extends React.PureComponent<RulerProps> implements Ru
   }
 
   private renderBackground() {
-    const { backgroundColor, lineColor, textColor } = this.props as Required<RulerProps>;
+    const { backgroundColor, lineColor, textColor } = this._getTheme(this.props.theme);
```

Comment:
seems like you have duplicated methods which is not [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) way
`_getTheme` in ruler
and the new one you created in Guides
`getCurrentTheme`

Create a public method in ruler and use it in both places